### ISSUE TITLE
Co-related Telemetry

### ISSUE THREAD

**Issue by ismaan1998** [Azure-Samples/Iot-Telemetry-Simulator#27]
Team,

We're using this telemetry simulator to simulate one of the machines which have some values like Motor Speed, Belt Speed, and more.

And **My Question is -**
Is there any way that we can simulate the co-related values? 
For example - 
1. The belt speed is co-related to Motor Speed. If the motor speed increases, how can we increase the belt speed as well?

Thanks

**Comment by fbeltrao** [comment]
Hi @ismaan1998,

How are you planning to define the motor speed? As a random value in a range?
What should the dependency between the motor and the belt speed? (i.e. motor speed + 1?)

Not that the simulator can do it, just want to understand your requirements.

**Comment by ismaan1998** [comment]
Yeah it could be - belt speed => 2* motor speed. Something like this.

**Comment by fbeltrao** [comment]
Thank you for the clarification. For now, we don't support such scenario. Closet would be to use two counter variables with different steps creating a linear progression,

**Comment by ismaan1998** [comment]
No worries, Thanks @fbeltrao for clarification. For now, we have written the custom code to simulate one of the machines in the production line. Closing this issue.
